app/Http/Controllers/ResultController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Result;
+use Illuminate\Http\Request;
+use App\Http\Resources\Answer;
+
+class ResultController extends Controller
+{
+    /**
+     * Display the specified resource.
+     *
+     * @param mixed $resultId
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index($resultId)
+    {
+        $result = Result::whereId($resultId)->answers();
```

Comment:
There also you should have pagination. So, please change the query and fetch the answers from the Answer model.